EveripediaNetwork/ep-ui
# Reverted tag suggestions

# Reverted Tag Suggestions

_This PR reverts tags suggestions_

## How should this be tested?

1. Navigate to edit wiki / create wiki page
2. Click on Edit Wiki Details button, and confirm the tag suggestions list has been reverted

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/3111
